### ISSUE TITLE
System.TimeoutException: The operation has timed out

### ISSUE THREAD

**Issue by nrbray** [canton7/SyncTrayzor#357]
```
Version: 1.1.15.0; Variant: Installed; Arch: Amd64
Path: C:\Program Files\SyncTrayzor\SyncTrayzor.exe
System.TimeoutException: The operation has timed out.
   at System.IO.Pipes.NamedPipeClientStream.Connect(Int32 timeout)
   at SyncTrayzor.Services.Ipc.IpcCommsClient.SendCommand(String command) in C:\projects\synctrayzor\src\SyncTrayzor\Services\Ipc\IpcCommsClient.cs:line 53
   at SyncTrayzor.Bootstrapper.Configure() in C:\projects\synctrayzor\src\SyncTrayzor\Bootstrapper.cs:line 123
   at Stylet.BootstrapperBase.Start(String[] args) in c:\projects\stylet\Stylet\BootstrapperBase.cs:line 81
   at System.Windows.Application.OnStartup(StartupEventArgs e)
   at System.Windows.Application.<.ctor>b__1_0(Object unused)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by canton7** [comment]
Thanks!

I've got a fix for this one lined up, but I haven't released it yet.

**Comment by HGRogers** [comment]
Any updates for this? My system just developed the same issue. 
Thanks

**Comment by canton7** [comment]
You can grab an intermediate build from here: https://ci.appveyor.com/project/canton7/synctrayzor/build/artifacts. I should release a version which fixes this soon...

**Comment by HGRogers** [comment]
That seems to have fixed it - thanks very much.
All the best

**Comment by canton7** [comment]
Fixed in 1.1.16.
